transcom/mymove
# MB-15610 - Add sit playwright tests

## [Jira ticket](https://dp3.atlassian.net/browse/MB-15610)

## Summary

This PR should introduce two testharness scenarios that create moves with shipments in SIT.

Something to pay extra attention to: I moved a bunch of `waitForPage` functions from the playwright/tests/office/servicescounseling/servicesCounselingTestFixture.js to playwright/tests/utils/officeTest.js since these pages are shared between all office users, and pages that implement `OfficePage` can add their own pages for specific user roles.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. check out the new testharness scenarios by visiting the testharness url
/testharness/list
new scenarios are named `HHGMoveIn200DaysSIT` and `HHGMoveIn200DaysSITWithPendingExtension`
2. run the new SIT playwright tests from a console window
`yarn playwright test playwright/tests/office/txo/sitUpdates.js`

### Frontend


### Backend


## Screenshots
